### DISCUSSION TITLE
How to build the static libs for macOS (M1 & Intel)

### DISCUSSION THREAD

**Discussion by StefanOltmann** [lopcode/vips-ffm#119] [Q&A]
The libs installed by Homebrew are dynamically typed, so I'll need to build static libs on my own. I have no experience with that.

I tried to follow https://github.com/libvips/libvips?tab=readme-ov-file#cheatsheet and also used ChatGPT, but I didn't end up with files.

Do you happen to know how I can build these from source for the macOS version of my app?

**Comment by lopcode** [comment]
I haven't tried so might not be much help - but the libvips folks are very helpful, so I think you can open a Discussion over there 🤞

**Reply by StefanOltmann** [reply]
Thank you for your quick response. I'll see if they can help.
